boards/xtensa/esp32s3/esp32s3-devkit/configs/toywasm/defconfig

```diff
@@ -71,7 +72,6 @@ CONFIG_PTHREAD_MUTEX_TYPES=y
 CONFIG_RAM_SIZE=114688
 CONFIG_RAM_START=0x20000000
 CONFIG_RR_INTERVAL=200
-CONFIG_SCHED_LPWORK=y
```

Comment:
Isn't Work queue needed anymore?

Reply:
it disappeared here because it's selected by ESP32S3_SPI_FLASH_SUPPORT_PSRAM_STACK.